iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -163,14 +163,6 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationMessageCellLay
 
         self.dataSource.apply(snapshot, animatingDifferences: false)
     }
-
-    func handle(isCentered: Bool) {
-        guard self.collectionLayout.showMessageStatus != isCentered else { return }
-
-        UIView.animate(withDuration: 0.2) {
-            self.collectionLayout.showMessageStatus = isCentered
-        }
-    }
```

Comment:
Showing the message status isn't handled by the layout anymore. handleIsCentered is no longer needed.